### ISSUE TITLE
IssuingCard missing expandable Number and CVC fields

### ISSUE THREAD

**Issue by jheth** [stripe/stripe-go#1081]
https://github.com/stripe/stripe-go/blob/6fa66eac3d47b497b26459ef913495ffd79182ac/issuing_card.go#L189

According to the docs, `cvc` and `number` are `expandable` fields but there are no struct fields for them to populate. 

My solution is to add the fields:

```
CVC        string     `json:"cvc"`
Number     string     `json:"number"`
```

I was hoping to create a pull request from github instead of forking the repo for this change.

**Comment by remi-stripe** [comment]
@jheth Thanks for the heads up and sorry for the missing fields. I was sure I built this in all our client libs but it seems I never finalized Go. I'll get this fixed soon.

**Comment by remi-stripe** [comment]
@jheth I just released 71.6.0 with the fix!

**Comment by jheth** [comment]
Fantastic. Thanks for the quick turn around!
